modules/virt-creating-vm-custom-template.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-vm-custom-template{context}"]
+= Creating a virtual machine from a customized template
+
+Some templates require additional parameters, for example, a PVC with a boot source. You can customize select parameters of a template to create a virtual machine (VM).
+
+.Procedure
+
+. Click *Virtualization* > *Catalog* in the side menu.
+
++
+Optional: Filter the templates by operating system or workload profile.
+
+. Click the template and then click *Customize VirtualMachine* to modify the template, for example, by specifying a boot source, the disk size, or a DataSource to clone.
+
+. Click *Review and create VirtualMachine*.
+
++
+Optional: Edit the parameters of the virtual machine.
```

Comment:
Since the procedure is about customizing the template to create the VM, I do not think that the edit step is optional. Suggestion: "Click *Review and create VirtualMachine* and edit the template parameters."